vmc/error_handler.go

```diff
@@ -73,7 +73,7 @@ func logVapiErrorData(message string, vAPIMessages []std.LocalizableMessage, vap
 
 	details := fmt.Sprintf(" %s: %s", message, printAPIError(apiError))
 	log.Printf("[ERROR]: %s", details)
-	return fmt.Errorf(details)
+	return fmt.Errorf("%s", details)
```

Comment:
one other idea that I used in the VCF provider `errors.New("...")`
it's neat for cases where you don't need an actual parametrized string template